src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -2572,38 +2518,28 @@ public function GetLastRevisionNumber()
      */
     public function AddNewRevisionFromDatabase($oFields, $oTablerecord, $postData, $sParentId = '')
     {
-        return $this->AddNewRevision_Execute($oFields, $oTablerecord, $postData, $sParentId);
+        return false;
     }
 
     /**
+     * @deprecated since 6.3.0 - revision management is no longer supported
+     *
      * saves the record with $postData
      * checks if postdata is valid and calls PostSaveHook after save.
      *
      * @param array $postData
      * @param bool  $bPreventMessage - if true the save success message will be suppressed
      *
-     * @return TCMSstdClass - stdclass from GetObjectShortInfo - returns false if revision could not be saved
+     * @return TCMSstdClass|bool - stdclass from GetObjectShortInfo - returns false if revision could not be saved
      */
     public function AddNewRevision(&$postData, $bPreventMessage = false)
     {
-        $postData = $this->PrepareDataForSave($postData);
-        $oPostTable = $this->GetNewTableObjectForEditor();
-        $oPostTable->Load($this->sId);
-        $iLastRevNr = $this->GetLastRevisionNumber();
-        $iNewRevNr = ($iLastRevNr + 1);
-        $oFields = &$this->oTableConf->GetFields($oPostTable);
-        $returnVal = $this->AddNewRevision_Execute($oFields, $oPostTable, $postData);
-        if (!empty($returnVal) && !$bPreventMessage) {
-            /** @var $oMessageManager TCMSMessageManager */
-            $oMessageManager = TCMSMessageManager::GetInstance();
-            $sConsumerName = TCMSTableEditorManager::MESSAGE_MANAGER_CONSUMER;
-            $oMessageManager->AddMessage($sConsumerName, 'TABLEEDITOR_REVISION_SAVED', array('sRevNr' => $iNewRevNr));
```

Comment:
The message `TABLEEDITOR_REVISION_SAVED` should also be deprecated.

Reply:
added